ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -90,43 +89,60 @@ function Get-sbRDSession {
                 Get-RDUserSession | Where-Object {
                     $_.UserName -like "*$user*"
                 }
-        }#foreach user
-    } else {
-        if ($IncludeSelf) {
-            Write-Verbose "Querying RD Session Collection for [$SessionState] sessions - including [$env:USERNAME]"
-            $sessions = Get-RDUserSession | Where-Object {
-                $_.SessionState -like $stateLookup.$SessionState`
-                    -and ( $_.IdleTime / 60000 ) -ge $MinimumIdleMins`
+            }#foreach user
+        }
+        else {
+            if ($IncludeSelf) {
+                Write-Verbose "Querying RD Session Collection for [$SessionState] sessions - including [$env:USERNAME]"
+                $sessions = Get-RDUserSession | Where-Object {
+                    $_.SessionState -like $stateLookup.$SessionState`
+                        -and ( $_.IdleTime / 60000 ) -ge $MinimumIdleMins`
+                }
+            }
+            else {
+                Write-Verbose "Querying RD Session Collection for [$SessionState] sessions"
+                $sessions = Get-RDUserSession | Where-Object {
```

Comment:
Only indent one level, like this:
```     
$sessions = Get-RDUserSession | Where-Object {
    $_.SessionState -like $stateLookup.$SessionState
    -and ( $_.IdleTime / 60000 ) -ge $MinimumIdleMins
    -and $_.UserName -ne "$env:USERNAME"
}           
```

Reply:
I think you can remove the backticks too

Reply:
Done - it allowed the formatter to correct the indentation as predicted.